pinot-segment-local/src/main/java/org/apache/pinot/segment/local/realtime/converter/RealtimeSegmentConverter.java

```diff
@@ -53,11 +55,13 @@ public class RealtimeSegmentConverter {
   private final List<String> _varLengthDictionaryColumns;
   private final boolean _nullHandlingEnabled;
 
-  public RealtimeSegmentConverter(MutableSegmentImpl realtimeSegment, String outputPath, Schema schema,
-      String tableName, TableConfig tableConfig, String segmentName, String sortedColumn,
-      List<String> invertedIndexColumns, List<String> textIndexColumns, List<String> fstIndexColumns,
-      List<String> noDictionaryColumns, List<String> varLengthDictionaryColumns, boolean nullHandlingEnabled) {
+  public RealtimeSegmentConverter(MutableSegmentImpl realtimeSegment, SegmentZKPropsConfig segmentZKMetadataConfig,
+      String outputPath, Schema schema, String tableName, TableConfig tableConfig, String segmentName,
+      String sortedColumn, List<String> invertedIndexColumns, List<String> textIndexColumns,
+      List<String> fstIndexColumns, List<String> noDictionaryColumns, List<String> varLengthDictionaryColumns,
+      boolean nullHandlingEnabled) {
     _realtimeSegmentImpl = realtimeSegment;
+    _segmentZKMetadataConfig = segmentZKMetadataConfig;
```

Comment:
rename variable to match with new name of class

Reply:
Ack